src/app.js

```diff
@@ -1,7 +1,26 @@
+//@ts-check
 import "dotenv/config";
 import express from "express";
+import errorHandler from "./middleware/error-handler.js";
 
-export const app = express();
-export const port = process.env.PORT || 4041;
+// controllers
+import accountRoutes from "./routes/account.js";
+
+const app = express();
+
+process.on("uncaughtException", err => {
+    console.log("UNCAUGHT EXCEPTION! 💥 Shutting down...");
```

Comment:
Not a deal breaker o, but for errors, I prefer to use `console.error` because we're logging an error message.

There are multiple console methods. A few of them are:

- console.log(...)
- console.error(...)
- console.warn(...)
- console.info(...)

Reply:
Oh thank you
I saw this somewhere and thought it's custom made, had no idea i can use